blog/2024-06-06-we-are-on-gpt-4o.mdx

```diff
@@ -0,0 +1,27 @@
+---
+slug: we-are-on-chatgpt-4o
+title: We are on GPT 4o 🚀
+description: Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model.
+authors: [jcizmar]
+tags: [chatgpt, tolgee, ai, tolgee-translator]
+---
+
+Great news! 🚀 Tolgee Translator is now backed by GPT-4o, the latest OpenAI's flagship model. It returns better results, and it is much less error-prone.
+
+<!--truncate-->
+
+## It fails less
+The GPT-3-turbo we used before sometimes failed since it could not provide results for some string language
+pairs. Due to this problem, sometimes the translation fails. The GPT 4o is more stable.
+
+## It understands the context better
+The GPT-4o better understands the context of the strings gathered from the Tolgee SDKs.
+So, it provides better results based on the context provided.
```

Comment:
word "provided" is 2x in the sentence. I'd suggest: This means that your translations will not only be more precise but also more nuanced, capturing the intended meaning more effectively.